corehq/apps/app_manager/forms.py

```diff
@@ -111,13 +134,30 @@ def __init__(self, *args, **kwargs):
 
         self.helper.label_class = 'col-sm-3 col-md-2'
         self.helper.field_class = 'col-sm-9 col-md-8 col-lg-6'
+        self.helper.field_style = 'ganggs'
         self.helper.form_text_inline = True
 
+        show_apk_version_select = kwargs.get('initial', {}).get('apk_prompt', 'off') != 'off'
+        show_app_version_select = kwargs.get('initial', {}).get('app_prompt', 'off') != 'off'
         self.helper.layout = crispy.Layout(
             crispy.Fieldset(
                 _("Manage Update Settings"),
-                crispy.Field('app_prompt'),
-                crispy.Field('apk_prompt'),
+                crispy.Field(
+                    'apk_prompt',
+                    # hide 'apk_version' depending on whether app_prompt is off or not
+                    onchange='$("#apk_version_id")[$("#id_apk_prompt").val() == "off"? "hide": "show"]()',
+                ),
+                crispy.Div(
+                    'apk_version',
+                    style=('' if show_apk_version_select else "display: none;"), css_id="apk_version_id"),
```

Comment:
Is this to just to set if it would be visible when page/form loads? 
Would be nice to just add a comment since this would work with the show/hide above but won't conflict with it.

Reply:
Yeah. Added comment